desktop/main.js

```diff
@@ -323,7 +334,20 @@ const mainWindow = () => {
                             {id: 'cut', role: 'cut'},
                             {id: 'copy', role: 'copy'},
                             {id: 'paste', role: 'paste'},
-                            {id: 'pasteAndMatchStyle', role: 'pasteAndMatchStyle'},
+                            {
+                                id: 'pasteAndMatchStyle',
+                                role: 'pasteAndMatchStyle',
+                                accelerator: 'Option+Shift+CmdOrCtrl+V',
+                            },
+                            {
+                                id: 'pasteAsPlainText',
+                                accelerator: PASTE_AS_PLAIN_TEXT_ACCELERATOR,
+                                click: () => {
+                                    // Insert the plain text from the clipboard
+                                    const text = clipboard.readText();
```

Comment:
Generally if I copy an image and then try to Paste, `macOS` will default to disable the menu. Here I can see we always show the menu.  Is there a way we can disable the menu in the Toolbar, and hide from the Context menu if there is no text to paste?